content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -7,75 +7,35 @@ description: "Open-source software developed by SBB CFF FFS and its integration
-  loop while synchronization is ON
-    alt any modification in OSRD (parent window)
-      OSRD (parent window)-->>OSRD (parent window): any modification
-      OSRD (parent window)->>NGE (osrd-nge iframe): iframe.contentWindow.postMessage(nge_light_model, "*")
-    else any modification in NGE (osrd-nge iframe)
-      NGE (osrd-nge iframe)-->>NGE (osrd-nge iframe): any modification <br>(hooks)
-      NGE (osrd-nge iframe)->>OSRD (parent window): parent.window.postMessage(nge_action, "*")
-      OSRD (parent window)-->>OSRD (parent window): convert NGE action into <br/>timetable update
-      OSRD (parent window)->>NGE (osrd-nge iframe): iframe.contentWindow.postMessage(nge_light_model, "*")
-    end
-  end
-```
-
-Since OSRD is the only source of truth (= data source), NGE has to be updated as the very last action, to be aligned with the timetable current state, and to get the updated microscopic calculations from OSRD.
-
-More precisely, when a change is made in NGE:
-
-```mermaid
----
-title: OSRD - NGE Communication when modification in NGE
----
-sequenceDiagram
-  autonumber
-  NGE (nge-app)-->NGE (osrd-nge): osrd-nge listens on hooks <br>from nge-app
-  NGE (osrd-nge)->>NGE (osrd-nge): (eventListeners) <br> callback triggered
-  NGE (osrd-nge)->>OSRD (front-end): (callback functions) <br>parent.window.postMessage(nge_action, "*")
-  OSRD (front-end)->>OSRD (front-end): (eventListeners) <br> convert nge_action <br> into timetable update
-  OSRD (front-end)->>OSRD (back-end): call for update timetable
-  OSRD (back-end)->>OSRD (front-end): update timetable
-  OSRD (front-end)->> OSRD (front-end): convert timetable into <br> nge_light_model
-  OSRD (front-end)->>NGE (osrd-nge): iframe.contentWindow.postMessage(nge_light_model, "*")
-  NGE (osrd-nge)->>NGE (osrd-nge): (eventListener) <br> onLoad triggered
-  NGE (osrd-nge)->>NGE (nge-app): (callback function) <br> onLoad(nge_light_model, ...)
-```
+![Concept diagram](osrd_nge_concept_diagram.png)
+
+NGE is then able to obtain the OSRD timetable as soon as a change is made on the OSRD side, and OSRD is able to obtain the changes made on the NGE side.
 
 #### 2 - Converters
 
-To overpass the semantic differences and make the data models fit, 2 converters have to be implemented:
-- **[OSRD -> NGE]** a converter that transforms an OSRD timetable into an NGE model
-- **[OSRD <- NGE]** an event handler, that transforms NGE action into OSRD database update on timetable
+To overcome semantic differences and adapt data models, two converters are implemented:
+- **[OSRD -> NGE]** a converter which transforms an OSRD timetable into an NGE model. The nodes are the remarkable points described by the circulations, and whose macroscopic information (position on the reticular) is stored in the database. OSRD train schedules, `TrainSchedule`, then represent cadenced train lines, `Trainrun`. A concept of cadenced train lines, will soon be implemented to allow conceptual convergence between OSRD and NGE.
```

Comment:
- The English translation for "point remarquable" is "operational point". The translation for "circulations" is "train schedule".
- Nodes may be track offsets, so maybe "waypoints" would be a better word?
- Maybe add "in NGE" after "represent cadenced train lines", to make it clear that "Trainrun" is a NGE-specific word?